### ISSUE TITLE
Straight vertical subpixel rendering e0b4 & e0b6 ./. e0b0 & e0b2

### ISSUE THREAD

**Issue by luebking** [ryanoasis/nerd-fonts#1547]
#### 🗹 Requirements

- [x] I have searched the [issues](https://github.com/ryanoasis/nerd-fonts/issues) for my issue and found nothing related and/or helpful
- [x] I have searched the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting) for help
- [x] I have searched the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki) for help

#### 🎯 Subject of the issue

**Experienced behavior:**
Subpixel rendering e0b4 & e0b6 (round powerline caps) is broken in comparism to e0b0 & e0b2

There's a red vertical bar on right end of e0b6 and a blue vertical bar on left end of e0b4
Happens with at least SauceCode and Monofur

The behavior can *somewhat* be mitigated by fumbling with the font sizes (heavily depends on TE and DPI, using urxvt and pixelsize I can precisely control this, but not fix it) and the LCD filter (lcdnone and lcdlegacy generally look "bad", but I can find a font size where it's just about fixed - but then there's an even amount of pixels and the glyph vertically off by one. lcddefault and lcdlight will always render a discolored bar - at least I could not fumble it away but for unreasonably -on 96 DPI- large fonts)

There's no such issue with the angular powerline caps, maybe fixed by https://github.com/ryanoasis/nerd-fonts/issues/156#issuecomment-1278487200 ?

**Expected behavior:**
Subpixel rendering e0b4 & e0b6 to be more like that of e0b0 & e0b2

**Example symbols:**
- e0b4
- e0b6

#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_
   - SauceCodeProNerdFont-Regular.ttf
   - MonofurNerdFont-Regular.ttf
- _Where did you get the file from (download link, self patched, source downloaded from link...)_
   - https://archlinux.org/packages/?q=nerd
- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_
   - urxvt
   - alacritty
   - xterm
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_
   - archlinux

**Comment by Finii** [comment]
Thanks for reporting!

The triangular things ... they were my test to see if the change I applied just to them breaks the rendering for some other users.
They were introduced with
* #1419

after some other different tries like
* #780

that fixed the vertical lines but added artifacts in other clients.

This is also related to
* #1490

because other issues become obvious there.

Thanks for pointing this out, and this has be be unified in some way or another, which will come with #1490.

Anyhow, this is kind of an 'unsolvable' problem because fixing for some will break it for others and we can only compromise where all can live with it. In fact I just turn on greyscale subpixel rendering as the resolution of nowadays displays is high enough (at least for my old eyes) - on Linux. Apple removed LCD subpixel rendering some years ago. And Windows, well, Windows is unfortunately kind of broken and noone seems to care (I can talk so much about Windows text rendering :sob:).

In conclusion
* Maybe switching to greyscale subpixel rendering helps you
* The round (and other) Powerline glyphs will get a thicker landing-strip soon (but possibly not wide enough to completely remove the effect (i.e. vertical lines) for all users

**Comment by Finii** [comment]
> on 96 DPI

ah I see...

**Comment by Finii** [comment]
From the aforementinoed PR

> **Implement this only for the filled triangles to see how it goes.**

Spreading this to the other glyphs now, as there were no complaints.

**Comment by Finii** [comment]
Added 7% landing strip to just these two

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/33f6689b-2f9f-4eed-a27c-1ee3c58dc2bd)
_green outline = new, filled = old_
